src/main/java/com/logistic/project/Application.java

```diff
@@ -2,7 +2,9 @@
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.cache.annotation.EnableCaching;
 
+@EnableCaching
```

Comment:
are you sure this is needed?

Reply:
AutoConfiguration. Or it's not required?